src/args.ts

```diff
@@ -56,6 +56,10 @@ export const args = Args.create(
         help: "Always equip the combat lover's locket, in order to get monsters inside quickly.",
         default: false,
       }),
+      backups: Args.flag({
```

Comment:
I would prefer to either call this `skipbackups`, or have true correspond to using backups.

Reply:
I've been meaning to add an arg to only avoid backuping tentacles, since I think the other backups are worth it in terms of MPA. But this arg is also useful.